packages/babel-make-styles/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@fluentui/babel-make-styles",
+  "version": "9.0.0-alpha.0",
+  "description": "A Babel plugin that performs build time transforms for @fluentui/make-styles",
+  "main": "lib/src/index.js",
+  "typings": "lib/src/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style",
+    "lint": "just-scripts lint",
+    "start-test": "just-scripts jest-watch",
+    "test": "just-scripts test"
+  },
+  "devDependencies": {
+    "@fluentui/eslint-plugin": "^1.1.0",
+    "@fluentui/react-make-styles": "^9.0.0-alpha.22",
+    "@fluentui/scripts": "^1.0.0",
+    "babel-plugin-tester": "^10.0.0",
+    "typescript": "4.2.4"
```

Comment:
⚠ This package depends on TS4 (_actually TS3.8+_) as `@linaria/babel` uses `import type` syntax that is not supported by our version 😯

I have the same problem for `@types/babel__helper-plugin-utils`, but I put it into `typings/babel-helper-plugin-utils/index.d.ts` temporary (these typings broke build for other packages, too 😐).

@ecraig12345 @Hotell do you have any suggestions how to avoid usage of a different TypeScript version there?

Reply:
@jdhuntington is working on a PR to upgrade to TS 4.1 (latest at the time the work was started) so hopefully this will be a non-issue soon.

Reply:
This will go in first probably, if this will happen, I will lease comments on TS4 upgrade PR. Thanks :)